pipeline-templates/job/code-build.yml

```diff
@@ -53,4 +53,9 @@ jobs:
     displayName: Publish Build Artifact
     inputs:
       targetPath: $(build.artifactstagingdirectory)/publish
-      artifactName: ${{ parameters.SolutionBaseName }}
\ No newline at end of file
+      artifactName: ${{ parameters.SolutionBaseName }}
+
+  - template: azure-pipelines-templates/build/step/nuget-pack.yml@das-platform-building-blocks
+    parameters:
+      DotNetStandardPackagesToPack: |
```

Comment:
@ChrisJWoodcock I assume this is the task to publish the shared nuget package to nuget

Reply:
Exactly